xcoll/scattering_routines/fluka/fluka_input.py

```diff
@@ -55,7 +55,7 @@ def _coll_dict(elements, names, dump=False):
                 elif ee.gap_R is not None:
                     nsig = ee.gap_R
                 half_gap = (ee._jaw_LU + ee._jaw_LD - ee._jaw_RU - ee._jaw_RD) / 4
-                offset   = (ee._jaw_LU + ee._jaw_LD + ee._jaw_RU + ee._jaw_RD) / 4
+                offset   = 0 # (ee._jaw_LU + ee._jaw_LD + ee._jaw_RU + ee._jaw_RD) / 4
```

Comment:
This is probably not the best approach. It would remove the manual offset in the tracking ([line 68 a.o. in track.py](https://github.com/xsuite/xcoll/blob/8b95488f5c47051ce1dcc47286ba14dad12fd2ac/xcoll/scattering_routines/fluka/track.py#L68)), such that the FLUKA input file always has the exact representation of the collimators in the line (hence including the offset).